conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalytics.java

```diff
@@ -89,28 +84,20 @@ public ConvivaAnalytics(BitmovinPlayer player,
                             ConvivaConfiguration config) {
         this.bitmovinPlayer = player;
         this.playerHelper = new BitmovinPlayerHelper(player);
+        this.context = context;
         this.config = config;
 
-        SystemInterface androidSystemInterface = AndroidSystemInterfaceFactory.buildSecure(context);
-        if (androidSystemInterface.isInitialized()) {
-            SystemSettings systemSettings = new SystemSettings();
-            systemSettings.allowUncaughtExceptions = false;
-
-            if (config.isDebugLoggingEnabled()) {
-                systemSettings.logLevel = SystemSettings.LogLevel.DEBUG;
-            }
-
-            SystemFactory androidSystemFactory = new SystemFactory(androidSystemInterface, systemSettings);
-            ClientSettings clientSettings = new ClientSettings(customerKey);
-
-            if (config.getGatewayUrl() != null) {
-                clientSettings.gatewayUrl = config.getGatewayUrl();
-            }
+        Map<String, Object> convivaSettings = new HashMap<>();
+        if (config.isDebugLoggingEnabled()) {
+            convivaSettings.put(ConvivaSdkConstants.LOG_LEVEL, ConvivaSdkConstants.LogLevel.DEBUG);
+        }
+        if (config.getGatewayUrl() != null) {
+            convivaSettings.put(ConvivaSdkConstants.GATEWAY_URL, config.getGatewayUrl());
+        }
 
-            this.client = new Client(clientSettings, androidSystemFactory);
+        com.conviva.sdk.ConvivaAnalytics.init(context, customerKey, convivaSettings);
```

Comment:
Did they really change the type name to the same we have 😂

Reply:
Yes. I can make the change to our name but then there will be ripple effects like documentation, release notes, breaking changes for clients, and a difference from iOS.

Reply:
No this is totally fine 🙂